pkg/virt-handler/virtwrap/manager.go

```diff
@@ -416,3 +427,125 @@ func (l *LibvirtDomainManager) KillVM(vm *v1.VM) error {
 	l.recorder.Event(vm, kubev1.EventTypeNormal, v1.Deleted.String(), "VM undefined")
 	return nil
 }
+
```

Comment:
@admiyo 

If you have something like that in mind:
```
virtwrap/manager.go #interface
virtwrap/libvirt/libvirt.go # libvirt impl in libvirt module
virtwrap/vdsm/vdsm.go # vdsm impl in vdsm module 
```
that sounds like a good start for me.